stable/jenkins/values.yaml

```diff
@@ -3,86 +3,57 @@
 # Declare name/value pairs to be passed into your templates.
 # name: value
 
-Master:
-  Name: jenkins-master
-  Image: "jenkinsci/jenkins"
-  ImageTag: "2.67"
-  ImagePullPolicy: "Always"
-  Component: "jenkins-master"
-  UseSecurity: true
-  AdminUser: admin
-# AdminPassword: <defaults to random>
-  Cpu: "200m"
-  Memory: "256Mi"
-# Set min/max heap here if needed with:
-# JavaOpts: "-Xms512m -Xmx512m"
-# JenkinsOpts: ""
-# JenkinsUriPrefix: "/jenkins"
-  ServicePort: 8080
-# For minikube, set this to NodePort, elsewhere use LoadBalancer
-# Use ClusterIP if your setup includes ingress controller
-  ServiceType: LoadBalancer
-# Master Service annotations
-  ServiceAnnotations: {}
-    #   service.beta.kubernetes.io/aws-load-balancer-backend-protocol: https
-# Used to create Ingress record (should used with ServiceType: ClusterIP)
-# HostName: jenkins.cluster.local
-# NodePort: <to set explicitly, choose port between 30000-32767
-  ContainerPort: 8080
-  SlaveListenerPort: 50000
-  LoadBalancerSourceRanges:
-  - 0.0.0.0/0
-# Optionally assign a known public LB IP
-# LoadBalancerIP: 1.2.3.4
-# Optionally configure a JMX port
-# requires additional JavaOpts, ie
-# JavaOpts: >
-#   -Dcom.sun.management.jmxremote.port=4000
-#   -Dcom.sun.management.jmxremote.authenticate=false
-#   -Dcom.sun.management.jmxremote.ssl=false
-# JMXPort: 4000
-# List of plugins to be install during Jenkins master start
-  InstallPlugins:
-      - kubernetes:0.11
-      - workflow-aggregator:2.5
-      - workflow-job:2.13
-      - credentials-binding:1.12
-      - git:3.4.0
-# Used to approve a list of groovy functions in pipelines used the script-security plugin. Can be viewed under /scriptApproval
-  # ScriptApproval:
-  #   - "method groovy.json.JsonSlurperClassic parseText java.lang.String"
-  #   - "new groovy.json.JsonSlurperClassic"
-# List of groovy init scripts to be executed during Jenkins master start
-  InitScripts:
-#  - |
-#    print 'adding global pipeline libraries, register properties, bootstrap jobs...'
-  CustomConfigMap: false
-# Node labels and tolerations for pod assignment
-# ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
-# ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
-  NodeSelector: {}
-  Tolerations: {}
+master:
+  name: master
```

Comment:
You are still keeping the name of the master component configurable. Doesn't make sense to me. Why would you ever change this?